servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/DefaultSingleAddressHttpClientBuilder.java

```diff
@@ -104,7 +104,7 @@ final class DefaultSingleAddressHttpClientBuilder<U, R> implements SingleAddress
     private static final StreamingHttpConnectionFilterFactory DEFAULT_IDLE_TIMEOUT_FILTER =
             new IdleTimeoutConnectionFilter(ofMinutes(5));
 
-    static final Duration SD_RETRY_STRATEGY_INIT_DURATION = ofSeconds(8);
+    static final Duration SD_RETRY_STRATEGY_INIT_DURATION = ofSeconds(2);
     static final Duration SD_RETRY_STRATEGY_MAX_DELAY = ofSeconds(256);
```

Comment:
Should we also lower the max cap down to 128 seconds (~2min)?

Reply:
That seems intuitively reasonable to me but some specific data would be helpful.

Reply:
what kind of data? I can only use rational thinking and ~5 min seems quite a long delay 😄

Reply:
I will merge as-is and we can discuss the max delay separately

Reply:
Sorry, I didn't see the reply. Data that would potentially be interesting would be what the conditions were around DNS overload scenarios, but I have a hard time imagining this would be a contributor to an otherwise healthy system falling over.

Reply:
https://github.com/apple/servicetalk/pull/2895